client/src/type/global.ts

```diff
@@ -0,0 +1,16 @@
+export type User = {
+  hash?: string;
+  username?: string;
+};
+
+export type Users = string[];
```

Comment:
Can make this name more specific, `Users` on a glance would imply `User[]`, if you are storing usernames in this you can call it Usernames, or UserHashes if you storing hashes.

Reply:
Yeah, `UserHashes` sounds suitable.